PythonPractices

```diff
@@ -0,0 +1,13 @@
+ for num in range (1,20):    
+     
+     if num%15 ==0:
```

Comment:
please install pylinter in your editor, so we can detect lint errors like this one.

you need to add whitespace at both sides
```suggestion
     if num%15 == 0:
```